### PR TITLE
Add integration test for cstor pool for sparse-mirrored-auto(dynamic)…

### PR DESCRIPTION
**Pull request by layog**
… spc with maxpool count 5.

**What this PR does / why we need it**: This PR adds integration test for spc with maxpool count 5.

**Which issue this PR fixes**: fixes openebs/openebs#2191

**Special notes for your reviewer**: I have not tested this locally as I do not have access to Kubernetes Cluster, but I guess this won't introduce any breaking changes.

### CODE REVIEW

File: integration-tests/cstor-pool/cstor_integration_test.go [hunk 1]
```diff
@@ -167,4 +167,95 @@ var _ = Describe("Integration Test", func() {
...
+			spcObject := &apis.StoragePoolClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "disk-claim-auto",
+				},
+				Spec: apis.StoragePoolClaimSpec{
+					Name:     "sparse-claim-auto",
+					Type:     "sparse",
+					MaxPools: 5,
+					PoolSpec: apis.CStorPoolAttr{
+						PoolType: "striped",
```
Comment (aswathkk) [inline integration-tests/cstor-pool/cstor_integration_test.go]: @layog PoolType should be `mirrored`

Comment (layog) [reply integration-tests/cstor-pool/cstor_integration_test.go]: Changed it to `mirrored`

File: integration-tests/cstor-pool/cstor_integration_test.go [hunk 2]
```diff
@@ -335,6 +335,100 @@ var _ = Describe("Integration Test", func() {
 			}
 			Expect(onlineCspCount).To(Equal(3))
 
+		})
+	})*/
```
Comment (aswathkk) [inline integration-tests/cstor-pool/cstor_integration_test.go]: Would you please check what */ is?
I think this is not required.

File: integration-tests/cstor-pool/cstor_integration_test.go [hunk 3]
```diff
@@ -335,6 +335,100 @@ var _ = Describe("Integration Test", func() {
...
+
+	// TODo: Add more test cases. Refer to following design doc
+	// https://docs.google.com/document/d/1QAYK-Bsehc7v66kscXCiMJ7_pTIjzNmwyl43tF92gWA/edit
+
+	When("We apply sparse-striped-auto spc yaml with maxPool count equal to 5 on a k8s cluster having at least 5 capable node", func() {
```
Comment (aswathkk) [inline integration-tests/cstor-pool/cstor_integration_test.go]: In the test description, change `sparse-striped-auto` to `sparse-mirrored-auto`

File: integration-tests/cstor-pool/cstor_integration_test.go (hunk 2 above)
Comment (layog) [reply integration-tests/cstor-pool/cstor_integration_test.go]: Yeah, this might have come up while rebasing

**Pull request merged**